docs/transports.md

```diff
@@ -1119,6 +1120,29 @@ logger.error(new Error("error"));
 logger.fatal("fatal");
 ```
 
+<a id="macfja-pino-fingers-crossed"></a>
+### @macfja/pino-fingers-crossed
+
+[@macfja/pino-fingers-crossed](https://github.com/MacFJA/js-pino-fingers-crossed) is a Pino v7+ transport that holds logs until a log level is reached, allowing to only have logs when it matters.
+
+```js
+import pino from "pino"
+import fingersCrossed, { enable } from "@macfja/pino-fingers-crossed"
```

Comment:
At the moment, my only feedback is to follow the same style as the other examples. That is, not ESM.

Reply:
My bad, I thought that both were okay.

Reply:
I'm just looking for consistency across the docs.

Reply:
Updated!

_(Sorry for the delay, I have to make my lib compatible with commonjs first)_